src/EFCore/Diagnostics/CoreLoggerExtensions.cs

```diff
@@ -448,10 +448,13 @@ private static string QueryCanceled(EventDefinitionBase definition, EventData pa
     ///     Logs for the <see cref="CoreEventId.QueryCompilationStarting" /> event.
     /// </summary>
     /// <param name="diagnostics">The diagnostics logger to use.</param>
+    /// <param name="context">The current <see cref="DbContext" />, or null if not known.</param>
```

Comment:
```suggestion
    /// <param name="context">The current <see cref="DbContext" />, or <see langword="null" /> if not known.</param>
```
And for the other ones